### ISSUE TITLE
Gearmenu behind player frame

### ISSUE THREAD

**Issue by ensidios** [RagedUnicorn/wow-classic-gearmenu#89]
![image](https://user-images.githubusercontent.com/62329082/145999236-38056653-bb5d-4d73-a516-dc818d414948.png)

slot-generated menu is behind player frame (LunaUnitFrames). All items behind this frame can't be selected. I didn't find any configuration for that issue. Is it possible to raise the level of Gearmenu?

Thank you very much!

**Comment by RagedUnicorn** [comment]
Hi @ensidios 

It is certainly possible to change the level of those elements. So far I just assumed people would pick a place where there is enough space for the menu to appear.

The problem I see with this is that I don't really know what level to put gearmenu on. When I developed the addon I tested mostly with the default ui and choose something that I deemed appropriate. In that case gearmenu is indeed on top of player frame. Then other addons modify this and a fight ensues on whos going to be ontop :x

So based on that I do not think I will adapt this currently. It would be possible for you to adapt this manually if absolutely needed.

```lua
/run GM_GearBarFrame_[someid]:SetFrameLevel(GM_GearBarFrame_[someid]:GetFrameLevel() + 1)
```

Cheers,
RagedUnicorn
